docs/7.x/config.md

```diff
@@ -13,7 +13,7 @@ Resource Name             | Resource Description
 `role`                    | User role (Enterprise only)
 `user`                    | Cluster user
 `token`                   | User tokens such as API keys
-`logforwarder`            | Forwarding logs to a remote rsyslog server
+`logforwarder`            | Forwarding logs to a remote rsyslog server!
```

Comment:
Why an exclamation mark here?

Reply:
that was an error, thanks for catching @a-palchikov !  it has been removed